{"uid":"7bbd407262eec15e","name":"test_check_email_text_on_verification_code_page[en]","fullName":"authorization.tests.test_verification_code.TestVerificationCodePassword#test_check_email_text_on_verification_code_page","historyId":"2e0101b02e4db35e186b4bec19f8241b","time":{"start":1724485952133,"stop":1724485952133,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x565327cf36ca <unknown>\n#1 0x5653279c4600 <unknown>\n#2 0x5653279fc485 <unknown>\n#3 0x5653279f82f8 <unknown>\n#4 0x565327a42ce8 <unknown>\n#5 0x565327a36643 <unknown>\n#6 0x565327a06d31 <unknown>\n#7 0x565327a0779e <unknown>\n#8 0x565327cbb25b <unknown>\n#9 0x565327cbf1f2 <unknown>\n#10 0x565327ca8615 <unknown>\n#11 0x565327cbfd82 <unknown>\n#12 0x565327c8d25f <unknown>\n#13 0x565327ce2e68 <unknown>\n#14 0x565327ce3040 <unknown>\n#15 0x565327cf249c <unknown>\n#16 0x7f834f894ac3 <unknown>","statusTrace":"@pytest.fixture(scope=\"function\")\n    @allure.title('Инициализация драйвера Chrome')\n    def browser():\n        options = Options()\n        # options.add_argument(\"--headless\")\n        options.add_argument('--window-size=1920,1080')\n>       browser = webdriver.Chrome(options=options)\n\nconftest.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py:45: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py:66: in __init__\n    super().__init__(command_executor=executor, options=options)\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:212: in __init__\n    self.start_session(capabilities)\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:299: in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f1d42d123d0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"session not created\",\"message\":\"session not created: Chrome failed to sta...\\\\n#14 0x565327ce3040 \\\\u003Cunknown>\\\\n#15 0x565327cf249c \\\\u003Cunknown>\\\\n#16 0x7f834f894ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\nE         (session not created: DevToolsActivePort file doesn't exist)\nE         (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nE       Stacktrace:\nE       #0 0x565327cf36ca <unknown>\nE       #1 0x5653279c4600 <unknown>\nE       #2 0x5653279fc485 <unknown>\nE       #3 0x5653279f82f8 <unknown>\nE       #4 0x565327a42ce8 <unknown>\nE       #5 0x565327a36643 <unknown>\nE       #6 0x565327a06d31 <unknown>\nE       #7 0x565327a0779e <unknown>\nE       #8 0x565327cbb25b <unknown>\nE       #9 0x565327cbf1f2 <unknown>\nE       #10 0x565327ca8615 <unknown>\nE       #11 0x565327cbfd82 <unknown>\nE       #12 0x565327c8d25f <unknown>\nE       #13 0x565327ce2e68 <unknown>\nE       #14 0x565327ce3040 <unknown>\nE       #15 0x565327cf249c <unknown>\nE       #16 0x7f834f894ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: SessionNotCreatedException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1724485951409,"stop":1724485951447,"duration":38},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"Инициализация драйвера Chrome","time":{"start":1724485952133,"stop":1724485952316,"duration":183},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x565327cf36ca <unknown>\n#1 0x5653279c4600 <unknown>\n#2 0x5653279fc485 <unknown>\n#3 0x5653279f82f8 <unknown>\n#4 0x565327a42ce8 <unknown>\n#5 0x565327a36643 <unknown>\n#6 0x565327a06d31 <unknown>\n#7 0x565327a0779e <unknown>\n#8 0x565327cbb25b <unknown>\n#9 0x565327cbf1f2 <unknown>\n#10 0x565327ca8615 <unknown>\n#11 0x565327cbfd82 <unknown>\n#12 0x565327c8d25f <unknown>\n#13 0x565327ce2e68 <unknown>\n#14 0x565327ce3040 <unknown>\n#15 0x565327cf249c <unknown>\n#16 0x7f834f894ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/motherhash/motherhash/conftest.py\", line 41, in browser\n    browser = webdriver.Chrome(options=options)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 212, in __init__\n    self.start_session(capabilities)\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 299, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"afterStages":[],"labels":[{"name":"subSuite","value":"2. Проверка наличия email, введённого ранее на странице входа"},{"name":"suite","value":"Verification Code"},{"name":"parentSuite","value":"authorization.tests"},{"name":"host","value":"fv-az1536-508"},{"name":"thread","value":"11926-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"authorization.tests.test_verification_code"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"language","value":"'en'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":6,"unknown":0,"total":7},"items":[{"uid":"a50a85c0c0b7c30e","reportUrl":"https://konstantingerasimovwork.github.io/motherhash/103//#testresult/a50a85c0c0b7c30e","status":"passed","time":{"start":1724426036477,"stop":1724426038497,"duration":2020}},{"uid":"135eb2566adb85ee","reportUrl":"https://konstantingerasimovwork.github.io/motherhash/102//#testresult/135eb2566adb85ee","status":"passed","time":{"start":1724423729980,"stop":1724423732011,"duration":2031}},{"uid":"84efb3064b15572e","reportUrl":"https://konstantingerasimovwork.github.io/motherhash/101//#testresult/84efb3064b15572e","status":"passed","time":{"start":1724418687521,"stop":1724418689560,"duration":2039}},{"uid":"a218423cb51a0e49","reportUrl":"https://konstantingerasimovwork.github.io/motherhash/100//#testresult/a218423cb51a0e49","status":"passed","time":{"start":1724407225975,"stop":1724407227982,"duration":2007}},{"uid":"e3a91f98ad04bcd7","reportUrl":"https://konstantingerasimovwork.github.io/motherhash/99//#testresult/e3a91f98ad04bcd7","status":"passed","time":{"start":1724405015735,"stop":1724405017765,"duration":2030}},{"uid":"f17739a41409bb72","reportUrl":"https://konstantingerasimovwork.github.io/motherhash/98//#testresult/f17739a41409bb72","status":"passed","time":{"start":1724402774644,"stop":1724402776690,"duration":2046}}]},"tags":[]},"source":"7bbd407262eec15e.json","parameterValues":["'en'"]}